tests/google-cloud-python/Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: all
+all:
+	docker build --no-cache .
```

Comment:
Also, build error:

[91mE: Failed to fetch http://httpredir.debian.org/debian/pool/main/b/blas/libblas-common_1.2.20110419-10_amd64.deb  Error reading from server. Remote end closed connection [IP: 128.31.0.66 80]

E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
[0mThe command '/bin/sh -c /scripts/install-apt-packages.sh' returned a non-zero code: 123
make: *** [build] Error 123